src/test/end-to-end/shared/shared-functions.ts

```diff
@@ -1,3 +1,63 @@
+import fs from 'fs';
+import { randomData } from './random-data';
+import path from 'path/posix';
+import os from 'os';
+
+const mockLocationFilePrefix = 'mock-location-';
+
+const locationJsonData = (id, name) => {
+    return [
+        {
+            'P&I ID': id,
+            'Court Desc': name,
+            'Welsh Court Desc': name,
+            Region: 'South East',
+            'Welsh Region': 'De-ddwyrain Lloegr',
+            Jurisdiction: 'Family; Civil',
+            'Welsh Jurisdiction': 'Llys Sifil; Llys Teulu',
+            Provenance: 'ListAssist',
+            'Provenance Location ID': '3482',
+            'Provenance Location Type': 'Venue',
+        },
+    ];
+};
+
+const generateTestLocationFields = (): [number, string] => {
+    // Generate a random test court ID between 200 and 1000000
+    const id = randomData.getRandomNumber(200, 1000000);
```

Comment:
I would up the starting number a bit just so it doesn't overlap when we get more courts.

Reply:
Now using a bigger starting number and a larger range for the court ID